tests/testthat/_snaps/emissions_profile.md

```diff
@@ -181,3 +181,45 @@
       24     0
       
 
+# At company level, three matched products with different `co2_footprint`, one missing benchmark, and one unmatched product yield the expected output
+
+    Code
+      missing_benchmark
+    Output
+      # A tibble: 8 x 4
+        companies_id grouped_by       risk_category value
+        <chr>        <chr>            <chr>         <dbl>
+      1 a            isic_4digit      high            0.2
+      2 a            isic_4digit      medium          0.2
+      3 a            isic_4digit      low             0.2
+      4 a            isic_4digit      <NA>            0.4
+      5 a            unit_isic_4digit high            0.2
+      6 a            unit_isic_4digit medium          0.2
+      7 a            unit_isic_4digit low             0.2
+      8 a            unit_isic_4digit <NA>            0.4
+
+---
+
+    Code
+      no_missing_benchmark
+    Output
+      # A tibble: 16 x 4
+         companies_id grouped_by       risk_category value
+         <chr>        <chr>            <chr>         <dbl>
+       1 a            all              high            0.4
+       2 a            all              medium          0.2
+       3 a            all              low             0.2
+       4 a            all              <NA>            0.2
+       5 a            tilt_sector      high            0.4
+       6 a            tilt_sector      medium          0.2
+       7 a            tilt_sector      low             0.2
+       8 a            tilt_sector      <NA>            0.2
```

Comment:
Here I explain the output not-affected by a missing benchmark. I focus on rows 1-4 where `grouped_by` is "all" but the same goes for each other level of `grouped_by`.

A total `value` of 1 is spread across 5 products, so the `value` for each product is `1 / 5 = 0.2`.

One product yields `NA` (because it was unmatched), so in the row (4) where `risk_category` is `NA` the `value` is `0.2`.

The other row (1) where `risk_category` is "high" shows the `value` of `0.4` -- corresponding to two matched products with a non-missing benchmark.

The other rows (2-3) each show a matched product with a non-missing benchmark, so each `value` is simply `0.2`.